server/src/main/java/io/crate/session/Sessions.java

```diff
@@ -78,6 +78,12 @@ public class Sessions {
         Setting.Property.Exposed
     );
 
+    public static final Setting<Integer> TEMP_ERROR_RETRY_COUNT = Setting.intSetting(
+        "node.sql.num_temp_error_retries",
```

Comment:
I'm not sure about the name or location, happy to hear better suggestions.

Reply:
Can you add a comment saying that this is intentionally undocumented and only used for tests?

I wonder if it would make more sense as a system property rather than a Setting?  Do we have any other settings which are for internal use only?

Reply:
> Do we have any other settings which are for internal use only?

We do have a couple of undocumented settings and even have a Setting.Property to indicate if a setting is exposed: https://github.com/crate/crate/commit/61218093086f182b5dc1125e4a5a7812d901c2ab

I'd keep it as setting instead of system property because this is something that I could imagine exposing eventually if we see this issue popping up in production systems.

Reply:
Added the comment